src/mavsdk/plugins/gimbal/gimbal_protocol_v2.cpp

```diff
@@ -86,11 +86,27 @@ Gimbal::Result GimbalProtocolV2::set_pitch_and_yaw(float pitch_deg, float yaw_de
                Gimbal::Result::Error;
 }
 
+void GimbalProtocolV2::set_angles_async(
+    float pitch_deg, float yaw_deg, float roll_deg, Gimbal::ResultCallback callback)
+{
+    // Sending the message should be quick and we can just do that straighaway.
+    Gimbal::Result result = set_angles(pitch_deg, yaw_deg, roll_deg);
+
+    if (callback) {
+        _system_impl.call_user_callback([callback, result]() { callback(result); });
+    }
+}
+
+Gimbal::Result GimbalProtocolV2::set_pitch_and_yaw(float pitch_deg, float yaw_deg)
+{
+    return set_angles(pitch_deg, yaw_deg, 0.0f);
+}
+
 void GimbalProtocolV2::set_pitch_and_yaw_async(
     float pitch_deg, float yaw_deg, Gimbal::ResultCallback callback)
 {
     // Sending the message should be quick and we can just do that straighaway.
-    Gimbal::Result result = set_pitch_and_yaw(pitch_deg, yaw_deg);
+    Gimbal::Result result = set_angles(pitch_deg, yaw_deg, 0.0f);
```

Comment:
Nice!

Reply:
If you do an interactive rebase you should be able to edit and amend that commit. Or you can just undo it again in a commit.